### PR TITLE
[Merged by Bors] - Compiler support oneflow to onnx

### PR DESCRIPTION
**Pull request by zzk0**

### CODE REVIEW

File: model_compiler/src/model_compiler/compilers/oneflow_model_file_to_onnx_model.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        dummy_inputs.append(flow.ones(shape, dtype=config.data_type))
+
+    model_module = utilities.load_module(source.script_path, 'Model')
+    model = model_module.Model()
+    model.load_state_dict(flow.load(source.model_path))
+    model.eval()
+    model_graph = Graph(model)
+    model_graph(*dummy_inputs)
+
+    tmpdirname = 'temp'
```
Comment (KellyZhang2020) [inline model_compiler/src/model_compiler/compilers/oneflow_model_file_to_onnx_model.py:63]: Use ``with-as``

File: model_compiler/src/model_compiler/models/sources/oneflow_model_file.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        input_script = value.get('script_path')
+        return isinstance(input_model, str) and path.isdir(input_model) and \
+            isinstance(input_script, str) and path.isfile(input_script)
+
+    @staticmethod
+    def accepts_env(env: Mapping[str, str]) -> bool:
+        input_model = env.get('ONEFLOW_PATH')
+        input_script = env.get('SCRIPT_PATH')
+        return isinstance(input_model, str) and path.isdir(input_model) and \
+            isinstance(input_script, str) and path.isfile(input_script)
```
Comment (KellyZhang2020) [inline model_compiler/src/model_compiler/models/sources/oneflow_model_file.py:41]: Add testcase

File: model_compiler/src/model_compiler/compilers/oneflow_model_file_to_onnx_model.py (hunk 1 above)
Comment (zzk0) [reply model_compiler/src/model_compiler/compilers/oneflow_model_file_to_onnx_model.py]: ok

File: model_compiler/src/model_compiler/models/sources/oneflow_model_file.py (hunk 2 above)
Comment (zzk0) [reply model_compiler/src/model_compiler/models/sources/oneflow_model_file.py:41]: testcase has added.

File: model_compiler/src/model_compiler/compilers/oneflow_model_file_to_onnx_model.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        def build(self, *args, **kwargs):
+            return self.model(*args, **kwargs)
+
+    dummy_inputs = []
+    for shape in config.input_shapes:
+        shape.insert(0, config.max_batch_size)
+        dummy_inputs.append(flow.ones(shape, dtype=config.data_type))
+
+    model_module = utilities.load_module(source.script_path, 'Model')
+    model = model_module.Model()
```
Comment (KellyZhang2020) [inline model_compiler/src/model_compiler/compilers/oneflow_model_file_to_onnx_model.py:59]: Use model = utilities.load_module(source.script_path, 'Model').Model()

Comment (zzk0) [reply model_compiler/src/model_compiler/compilers/oneflow_model_file_to_onnx_model.py]: ok

**Pull request closed without merging**